collect_app/src/main/java/org/odk/collect/android/formlists/blankformlist/BlankFormListActivity.kt

```diff
@@ -57,7 +55,7 @@ class BlankFormListActivity : LocalizedActivity(), OnFormItemClickListener {
 
         val list = findViewById<RecyclerView>(R.id.form_list)
         list.layoutManager = LinearLayoutManager(this)
-        list.addItemDecoration(RecyclerViewUtils.verticalLineDivider(this))
+        list.addItemDecoration(org.odk.collect.lists.RecyclerViewUtils.verticalLineDivider(this))
```

Comment:
You can import `RecyclerViewUtils` and remove `org.odk.collect.lists` here.
The same with `EmptyListView`, `MultiSelectAdapter` etc below.